daemon/cluster/cluster.go

```diff
@@ -519,24 +519,24 @@ func (c *Cluster) Leave(force bool) error {
 	}
 
 	if node.Manager() != nil && !force {
-		msg := "You are attempting to leave cluster on a node that is participating as a manager. "
+		msg := "You are attempting to leave the swarm on a node that is participating as a manager. "
 		if c.isActiveManager() {
 			active, reachable, unreachable, err := c.managerStats()
 			if err == nil {
 				if active && reachable-2 <= unreachable {
 					if reachable == 1 && unreachable == 0 {
-						msg += "Removing the last manager will erase all current state of the cluster. Use `--force` to ignore this message. "
+						msg += "Removing the last manager erases all current state of the swarm. Use `--force` to ignore this message. "
 						c.Unlock()
 						return fmt.Errorf(msg)
 					}
-					msg += fmt.Sprintf("Leaving the cluster will leave you with %v managers out of %v. This means Raft quorum will be lost and your cluster will become inaccessible. ", reachable-1, reachable+unreachable)
+					msg += fmt.Sprintf("Leaving the swarm will leave you with %v managers out of %v. This means Raft quorum will be lost and your swarm becomes inaccessible. ", reachable-1, reachable+unreachable)
```

Comment:
suggest:
Removing this node leaves %v managers out of %v. Without a Raft quorum your cluster will be inaccessible.